guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -126,10 +115,6 @@ endif::[]
 +
 [options="nowrap" subs="quotes,attributes"]
 ----
-ifdef::satellite[]
-repo --name=rhel --baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7.7/x86_64
-repo --name=sat --baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/sat-capsule/{ProductVersion}/os
-endif::[]
```

Comment:
You can't just drop these repos, they should be replaced with RHEL 8 equivalents.

Reply:
Ohk. I will try to get it from other sources.

Reply:
that's easy, on modern (pulp3) based installs, those paths just don't exist. neither for el7 nor el8. so I'd leave them as broken as they are now (so use el7) and fix that in a different ticket "properly" (mind you, I don't know what properly means in this case, but that's out of scope for this PR)

Reply:
Noted @evgeni 
Will keep this change as it is.